src/renderer/components/commands-address-bar.tsx

```diff
@@ -110,6 +111,15 @@ export class AddressBar extends React.Component<
     this.setState({ value: event.target.value });
   }
 
+  public handleBlur(event: React.FocusEvent<HTMLInputElement>) {
+    const { gistId } = this.props.appState;
+    const url = urlFromId(gistId);
+
+    if (event && url) {
```

Comment:
@aryanshridhar:

Is `event` ever falsy here?

Reply:
I don't think so , but since a event is being passed in the function parameter , I felt it better to pass it in the definition as well . (Will remove , if it doesn't feel right)